test/unit/search/fm_index_cursor/bi_fm_index_cursor_test.cpp

```diff
@@ -7,5 +7,41 @@
 
 #include "bi_fm_index_cursor_test_template.hpp"
 
+#include <seqan3/range/views/char_to.hpp>
+#include <seqan3/range/views/to.hpp>
+
+#include <string_view>
```

Comment:
stl includes before seqan includes.
Also please include the template header at the end